development/webpack/utils/plugins/SelfInjectPlugin/index.ts

```diff
@@ -144,24 +144,11 @@ export class SelfInjectPlugin {
 
     // generate the new self-injecting source code:
     const newSource = new ConcatSource();
-    // wrapped in a new lexical scope so we don't pollute the global namespace
-    newSource.add(`{`);
-    newSource.add(`let d=document,s=d.createElement('script');`);
-    newSource.add(`s.textContent=`);
+    // TODO: I removed sourcemap stuff so I could move forward
+    // TODO: but we might need to circle back to this
+    newSource.add(`document.INJECT_ONCE(`);
     newSource.add(this.escapeJs(source + sourceMappingURLComment));
```

Comment:
I assume that the the previous injection pattern doesn't work at this point because `source ` here hasn't been wrapped by lavamoat yet?